### PR TITLE
Add rendering apps to document types & CSVs

### PR DESCRIPTION
**Pull request by tijmenb**
This adds the rendering app of a document type to the document type page and to the CSV. We'll use this to check which applications we have to update for the first iterations in the navigation team.

The data is fetched from production and copied here because there is no API available that can do this accurately. The search API contains most of the information, but there's some content missing or incorrect.

There are some interesting issues in data (coming_soon being rendered by both whitehall-frontend and government-frontend).

https://trello.com/c/mKmejPHS

### CODE REVIEW

File: spec/fixtures/document-types-export.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Name,Docs URL,Number of pages,Example URL,Navigation document type
-announcement,https://docs.publishing.service.gov.uk/document-types/announcement.html,10439,https://www.gov.uk/government/news/credit-reports-available-online-for-all-consumers,other
-answer,https://docs.publishing.service.gov.uk/document-types/answer.html,737,https://www.gov.uk/apply-apprenticeship,guidance
-asylum_support_decision,https://docs.publishing.service.gov.uk/document-types/asylum_support_decision.html,0,,other
-authored_article,https://docs.publishing.service.gov.uk/document-types/authored_article.html,249,https://www.gov.uk/government/speeches/i-want-an-exit-that-will-work-for-all-of-us-article-by-theresa-may,other
+Name,Docs URL,Number of pages,Example URL,Rendering apps,Navigation document type
```
Comment (fofr) [inline spec/fixtures/document-types-export.csv]: Rendering app? Or can it be plural?

**Comment by fofr** [comment]
Looks good!

screen shot 2017-10-18 at 17 18 31

File: data/rendering-apps.yml [hunk 2]
```diff
@@ -0,0 +1,489 @@
+# Generated by running this in the content store console:
+# y ContentItem.distinct(:nameocument_type).map { |document_type| { name: document_type, apps: ContentItem.where(document_type: d).distinct(:rendering_app)
```
Comment (fofr) [inline data/rendering-apps.yml]: `nameocument_type`, guessing that's a typo?

Comment (tijmenb) [reply data/rendering-apps.yml]: The `y x` will output `x` as YAML!

You're right on the rest, I did some super dodgy copy pasting! PR upcoming.

Comment (fofr) [reply data/rendering-apps.yml]: Yeah, the `y` command is super useful.

Comment (suzannehamilton) [reply data/rendering-apps.yml]: TIL! 🎉

**Pull request merged**